models/br_tse_eleicoes/code/DEPENDENCIES.md

```
@@ -0,0 +1,192 @@
+### Outras
+
+| Table | Builder | Raw input | Phase 2 / 3 | Upstream tables | Utils |
+|---|---|---|---|---|---|
+| `vagas` | `sub/vacancies.py::build_vagas(ano)` (1994–2024, even years) | `consulta_vagas/consulta_vagas_{ano}/consulta_vagas_{ano}_{uf}` | — | — | `clean_election_type`, `clean_string`, `helpers.*` |
+| `dicionario` | static — not built by the Python pipeline | — | — | — | — |
+
+---
+
+## Cross-table dependency graph
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)